app/poros/direction.rb

```diff
@@ -0,0 +1,10 @@
+class Direction
+  attr_reader :travel_distance,
+              :directions_list
+
+    def initialize(data)
+      @directions_list = data[:maneuvers].map do |direction_instruction|
+        direction_instruction[:narrative]
+      end
+    end
+end
```

Comment:
I'd avoid putting a lot of logic in your initialize method. You could do something like this: 

```ruby
class Direction
  attr_reader :travel_distance, # do you need this?
              :directions_list

    def initialize(data)
      @directions_data = data[:maneuvers].map do |direction_instruction|
        direction_instruction[:narrative]
      end
    end
   def directions_list
         @directions_data.map do |direction_instruction|
                direction_instruction[:narrative]
          end
   end 
end
```